opensrp-giz-malawi/src/main/java/org/smartregister/giz/presenter/AdvancedSearchPresenter.java

```diff
@@ -24,9 +24,12 @@ protected AdvancedMatrixCursor getRemoteLocalMatrixCursor(AdvancedMatrixCursor m
         String query = getView().filterAndSortQuery();
         Cursor cursor = getView().getRawCustomQueryForAdapter(query);
         if (cursor != null && cursor.getCount() > 0) {
-            AdvancedMatrixCursor remoteLocalCursor = new AdvancedMatrixCursor(new String[]{DBConstants.KEY.ID_LOWER_CASE, DBConstants.KEY.RELATIONALID, DBConstants.KEY.FIRST_NAME, DBConstants.KEY.LAST_NAME, DBConstants.KEY.DOB, DBConstants.KEY.ZEIR_ID});
+            AdvancedMatrixCursor remoteLocalCursor = new AdvancedMatrixCursor(
+                    new String[]{DBConstants.KEY.ID_LOWER_CASE, DBConstants.KEY.RELATIONALID, DBConstants.KEY.FIRST_NAME, DBConstants.KEY.LAST_NAME, DBConstants.KEY.DOB, DBConstants.KEY.MER_ID});
```

Comment:
Does the column `DBConstants.KEY.ID_LOWER_CASE` exist in the DB?

Reply:
Fixed

Reply:
To test this just search and check the logs for the query executed?